docs/reference/content/index.md

```diff
@@ -8,10 +8,12 @@ type = "index"
 
 Welcome to the MongoDB Reactive Streams Java driver documentation hub.
 
-The Reactive Streams implementation is built upon the 
-[`MongoDB Async Driver`]({{< asyncDriver >}}) and we mirror their API. 
-For reference documentation please see the async driver documentation.
+The Reactive Streams implementation is built upon the callback-based [`MongoDB Async Driver`]({{< asyncDriver >}}) and mirrors its API. 
+For reference documentation please see the asynchronous driver documentation.
 
+Note that while the callback-based asynchronous driver is now deprecated, the MongoDB Reactive Streams Java 
+driver will continue to be supported. Once the callback-based driver has been removed, the MongoDB Reactive Streams Java 
+driver will be fully documented in its place   
```

Comment:
I don't understand why we can't fully document the Reactive driver now.

Reply:
We could, but it will take a lot more time than this PR.

Reply:
Also, those docs will be moved soon to the Java driver github pages, so any links people create to docs we add now will be stale in a few months.